src/format/opamSysPkg.ml

```diff
@@ -67,6 +67,41 @@ let string_of_status sp =
     (Set.to_string sp.s_available)
     (Set.to_string sp.s_not_found)
 
+let combine_status st st' =
+  Set.Op.{
+    s_available = st.s_available ++ st'.s_available;
+    s_not_found = st.s_available ++ st'.s_not_found;
+  }
+
+(* System package availability *)
+type availability_mode =
+  | Available of Set.t
+  | Suppose_available
+  | No_depexts
```

Comment:
Looking at the only places where this value is created (only when the checks raised an exception), i feel like this case is redundant with `Suppose_available` and doesn't add any value but confusion to the variant
```suggestion
```

Reply:
I'll let @rjbou make the final call :)